source/js/RecipeInfo.js

```diff
@@ -0,0 +1,283 @@
+class RecipeInfo extends HTMLElement {
+  constructor() {
+    super();
+    let shadow = this.attachShadow({ mode: "open" });
+  }
+
+  set data(data) {
+    const style = `
+        .recipe-info {
+          margin-left: 25vw;
+          margin-right: 25vw;
+          width: 50vw;
+          // display: grid;
+          // grid-template-rows: [top] 50% [image-bottom] 1.5em [title-bottom] 1.5em [info-bottom]  [bottom];
+          // grid-template-columns: [left] auto [right];
+    
+          background: #FFF6EC;
+        }
+    
+        .thumbnail-photo {
+          height: 50vh;
+          object-fit: cover;
+          width: 50vw;
+        }
+
+        .title {
+          text-align: center;
+          font-size: 3vh;
+          font-weight: bolder;
+        }
+
+        .rating-time {
+          padding-left: 1vw;
+          padding-right: 1vw;
+          display: grid;
+          grid-template-rows: [top] auto [bottom];
+          grid-template-columns: [left] 50% [middle] 50% [right];
+          border: 1px solid orange;
+        }
+
+        .star-image {
+          height: 5vh;
+          width: 5vw;
+          float: right;
+        }
+
+        .directions {
+          text-align: center;
+          font-size: 2.5vh;
+          font-style: italic;
+          font-weight: bold;
+          padding: none;
+        }
+
+        button {
+          height: 5vh;
+          width: 10vw;
+        }
+        `;
+
+    const styleElem = document.createElement("style");
```

Comment:
Codacy has a fix for the issue: [The variable `styleElem` should be named `styleElement`. A more descriptive name will do too.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8447291)
```suggestion
    const styleElement = document.createElement("style");
```